test/parallel/test-fs-lchown.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const common = require('../common');
```

Comment:
If there are other tests you think we should add to `lchown`, let me know and we'll make `good-first-issue`s out of them

Reply:
We're pretty light on `chown()`, `fchown()`, and `lchown()` happy path tests in general. My guess is because of platform support and getting the `uid` and `gid` parts right on the CI.